pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "symphony_bdk_python"
-version = "2.3.dev0"
+version = "2.4.dev0"
```

Comment:
since 2.4 is ready to release, should we not use 2.5.dev0 instead ?

Reply:
You're right. I don't remember why we kept using the same version as the release in main, with "dev" suffix. I corrected it in the last commit.